core/tests/test_utils.py

```diff
@@ -140,12 +157,26 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Adds a suffix to the UNICODE_TEST_STRING
```

Comment:
This is not useful to a caller. The point of this method is to supply a test string that has unicode characters in it (in order to check that things still work properly even if non-ascii characters are involved); the docstring should reflect this.